lib/xlsx/xform/book/workbook-pivot-cache-xform.js

```diff
@@ -0,0 +1,29 @@
+const BaseXform = require('../base-xform');
+
+class WorkbookPivotCacheXform extends BaseXform {
+  render(xmlStream, model) {
+    xmlStream.leafNode('pivotCache', {
+      cacheId: model.cacheId,
+      'r:id': model.rId,
+    });
+  }
+
+  parseOpen(node) {
+    if (node.name === 'pivotCache') {
+      this.model = {
+        cacheId: node.attributes.cacheId,
+        rId: node.attributes['r:id'],
+      };
+      return true;
+    }
+    return false;
```

Comment:
Not so important, only a little bit readability  improvement 😄 

```suggestion
    if (node.name !== 'pivotCache') {
       return false;
    }
    this.model = {
      cacheId: node.attributes.cacheId,
      rId: node.attributes['r:id'],
    };
    return true;
```

Reply:
I see the pattern you points towards, and I agree in general! However, this is copy paste from elsewhere in ExcelJS. So I tried to stay consistent with @guyonroche's pattern of returning `true` first and `false` later. However, since pivot table doesn't handle "read" for now, this code won't get run anyways. :)

Reply:
it's just a detail, nevermind